lib/dart/lib/src/browser/t_web_socket.dart

```diff
@@ -71,19 +73,19 @@ class TWebSocket implements TSocket {
     }
 
     _socket = WebSocket(url.toString());
-    _socket.onError.listen(_onError);
-    _socket.onOpen.listen(_onOpen);
-    _socket.onClose.listen(_onClose);
-    _socket.onMessage.listen(_onMessage);
+    _socket!.onError.listen(_onError);
```

Comment:
I'm personally not such a big fan of overriding null checks via `!` because there is a reason why these checks exist in the first place. It's nice when code runs during the shiny weather, but it should not produce strange, hard to reproduce and hard to spot errors under heavy conditions.